marlowe/src/Language/Marlowe/Client.hs

```diff
@@ -528,7 +529,7 @@ mkMarloweStateMachineTransition params SM.State{ SM.stateData=MarloweData{..}, S
                         finalBalance = totalIncome P.- totalPayouts
                         in (outputsConstraints, finalBalance)
             -- TODO Push this use of time further down the code
-            let range = TimeSlot.slotRangeToPOSIXTimeRange $ Interval.interval minSlot maxSlot
+            let range = TimeSlot.slotRangeToPOSIXTimeRange def $ Interval.interval minSlot maxSlot
```

Comment:
I guess we need to audit all uses of the default value, right? Since if you're not using the same one as the emulator, you're going to have a bad time!